backend/experiment/actions/base_action.py

```diff
@@ -17,35 +19,34 @@ class BaseAction(object):
     Example:
         ```python
         class CustomAction(BaseAction):
-            ID = "CUSTOM"
+            view = "CUSTOM"
```

Comment:
I agree that using "view" as the property name is easier to understand. But what effect will this have on the Frontend? 

`action_dict['view'] = self.view`

Will this line mean nothing has to change in the frontend (apart from the style-related logic)?

Reply:
Yep, the frontend has always referring to the action types as "view".